e2e-tests/tfm/cypress/e2e/journeys/case-amendments/6. cover-end-date.spec.js

```diff
@@ -108,79 +106,52 @@ context('Amendments - Cover End Date', () => {
-      cy.url().should('contain', 'request-date');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'request-approval');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'amendment-options');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'cover-end-date');
-      amendmentsPage.amendmentCoverEndDateDayInput().clear().focused().type(dateConstants.todayDay);
-      amendmentsPage.amendmentCoverEndDateMonthInput().clear().focused().type(dateConstants.todayMonth);
-      amendmentsPage.amendmentCoverEndDateYearInput().clear().focused().type(dateConstants.todayYear);
-      amendmentsPage.continueAmendment().click();
-
-      cy.url().should('contain', 'check-answers');
-
-      amendmentsPage.amendmentAnswerBankRequestDate().should('contain', dateConstants.todayDay);
-      amendmentsPage.amendmentAnswerRequireApproval().should('contain', 'Yes');
-      amendmentsPage.amendmentAnswerCoverEndDate().should('contain', dateConstants.todayDay);
-      amendmentsPage.amendmentAnswerIsUsingFacilityEndDate().should('not.exist');
-    });
+  it('should continue to the `Check answers` page if the cover end date is valid and only the cover end date is to be changed', () => {
+    cy.login(PIM_USER_1);
+    const facilityId = dealFacilities[0]._id;
+    cy.visit(relative(`/case/${dealId}/facility/${facilityId}`));
 
-    it('should continue to the `Enter the facility value` page if the cover end date is valid, the facility value also needs changing and the facility end date FF is disabled', () => {
-      cy.login(PIM_USER_1);
-      const facilityId = dealFacilities[0]._id;
-      cy.visit(relative(`/case/${dealId}/facility/${facilityId}`));
-
-      facilityPage.facilityTabAmendments().click();
-      amendmentsPage.continueAmendmentButton().click();
-      cy.url().should('contain', 'request-date');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'request-approval');
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'amendment-options');
-      amendmentsPage.amendmentFacilityValueCheckbox().click();
-      amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'cover-end-date');
-      amendmentsPage.amendmentCoverEndDateDayInput().clear().focused().type(dateConstants.todayDay);
-      amendmentsPage.amendmentCoverEndDateMonthInput().clear().focused().type(dateConstants.todayMonth);
-      amendmentsPage.amendmentCoverEndDateYearInput().clear().focused().type(dateConstants.todayYear);
-      amendmentsPage.continueAmendment().click();
-
-      cy.url().should('contain', 'facility-value');
-    });
-  }
+    facilityPage.facilityTabAmendments().click();
+    amendmentsPage.continueAmendmentButton().click();
+    cy.url().should('contain', 'request-date');
+    amendmentsPage.continueAmendment().click();
```

Comment:
could these be split into different it statements? It is easier to see where it is failing if something fails in the middle here

Reply:
I don't think this can be split out anymore - this is checking the next stage after the previous 'it' statement (it has to get up to the section it's testing first which requires clicking through lots of the pages first which is why it looks long). I've kept it consistent with the rest of the file.

Reply:
tech debt ticket for this please